### PR TITLE
Create addition.c

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** NONE

**Changes:** Added program to add 2 numbers **without any operator**

### CODE REVIEW

File: code/unclassified/src/addition_of_2_numbers_without_any_operator/addition.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+int main() 
+{
+    // test cases
+    int i, j;
+    for(i = 1; i <= 3; i++)
+    {
+        for(j = 1; j <= 3; j++)
+        {
+            printf("%d + %d = %d %c", i, j, add(i, j), '\n');
```
Comment (arnavb) [inline code/unclassified/src/addition_of_2_numbers_without_any_operator/addition.c]: Why not just directly print the newline?

Comment (tbhaxor) [reply code/unclassified/src/addition_of_2_numbers_without_any_operator/addition.c]: ohk i will :+1:

**Comment by tbhaxor** [comment]
@arnavb done :+1:

**Comment by tbhaxor** [comment]
@arnavb do i have to modify it again ;_;

**Comment by arnavb** [comment]
@tbhaxor You don't have to, but if you want to... (I approved because it was just suggestion)

**Comment by tbhaxor** [comment]
ohk then i wont :laughing:

**Comment by tbhaxor** [comment]
@AdiChat  please review this PR too

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**